cyences_app_for_splunk/default/savedsearches.conf

```diff
@@ -1734,6 +1793,101 @@ action.summary_index._name = cyences
 action.summary_index.category = O365
 action.summary_index.alert_name = O365 - Daily Login Failure
 
+[O365 - Exchange Rule]
+disabled = 1
+enableSched = 1
+alert.track = 1
+alert.severity = 4
+alert.suppress = 0
+counttype = number of events
+quantity = 0
+relation = greater than
+cron_schedule =	29 * * * *
+description = This alert will show the alerts for Exchange Rule. \
+\
+Data Collection - Office 365 management activity data (Splunk Add-on for Office 365). 
+dispatch.earliest_time = -85m@s
+dispatch.latest_time = -25m@s
+display.general.type = statistics
+display.page.search.tab = statistics
+display.page.search.mode = fast
+request.ui_dispatch_app = cyences_app_for_splunk
+request.ui_dispatch_view = search
+search = `cs_o365` sourcetype="o365:management:activity" Workload=SecurityComplianceCenter Operation=AlertTriggered \
+| dedup AlertId \
+| spath input=Data \
+| table _time Name Severity AlertType Category f3u lon wl ad \
+| `cs_o365_exchange_rule_filter`
+action.summary_index = 1
+action.summary_index._name = cyences
+action.summary_index.category = O365
+action.summary_index.alert_name = O365 - Exchange Rule
+
+[O365 - Hourly Increase In Emails Over Baseline]
```

Comment:
Instead of putting it under the O365 category, let's create a new category `Email`.
So, the alert name will the `Email - Hourly Increase In Emails Over Baseline`

For now, the query is okay. We'll update the search query when we've a customer with other email service then O365.

Reply:
You need to update the `Overview` dashboard for the new category of email. Or include them with the `O365`.